File: PracticeAndroid/Andorid_Kotlin/Liabrary/MyLiabraryProject/app/src/main/java/com/example/myliabraryproject/MainActivity.java
package com.example.myliabraryproject;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.toastlibrary.Toaster;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Toaster.showToast(this,"Hello Programmers");
    }
}

File: PracticeAndroid/Andorid_Kotlin/CustomLib/TestApplication/app/src/main/java/com/example/testapplication/MainActivity.kt
package com.example.testapplication

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.toastlibrary.CameraXActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        intent = Intent(this, CameraXActivity::class.java)
        startActivity(intent)
    }
}